packages/core/src/browser/keyboard/browser-keyboard-layout-provider.ts

```diff
@@ -59,7 +59,11 @@ export class BrowserKeyboardLayoutProvider implements KeyboardLayoutProvider, Ke
     }
 
     @postConstruct()
-    protected async initialize(): Promise<void> {
+    protected initialize(): void {
+        this.doInit();
+    }
```

Comment:
Just for the sake of being consistent but [you already renamed a previous post constructor to `init`](https://github.com/eclipse-theia/theia/pull/12425/files#diff-7786689c4e73bd3c218a39af4560a369240cb1f336e1d4c968c9bffd8d378aa7R199) so I think we could do it here too.